site-configs/ncrc/config.site

```diff
@@ -29,10 +29,10 @@ test -z "$CC" && CC=cc
 test -z "$FC" && FC=ftn
 
 # Compile/Link flags
-test -z "$CFLAGS" && CFLAGS="-xSSE2"
```

Comment:
Chris, The Intel default optimization flag is -O2. The GNU default optimization flag is -O0.
So we need to add the -O2 flag whenever we use gcc. Thats for both CFLAGS and FCFLAGS.